rainier-core/src/main/scala/com/stripe/rainier/core/Events.scala

```diff
@@ -0,0 +1,32 @@
+package com.stripe.rainier.core
+
+object Events {
+    def observe[T](seq: Seq[T], dist: Distribution[T]): RandomVariable[Generator[T]] =
```

Comment:
IRL we discussed having these be something like `def observe[T, D <: Distribution[T]](seq: Seq[T], dist: D): RandomVariable[D]`